### ISSUE TITLE
SCR modes for GTA III and VC

### ISSUE THREAD

**Issue by x87** [sannybuilder/dev#147]
A bit off topic, but I wanted to ask if there will be SRC modes for GTA 3 and GTA VC? I like working with this mode nowadays, and the library is great for SRC.

_Originally posted by @wmysterio in https://github.com/sannybuilder/dev/discussions/136#discussioncomment-844113_

**Comment by wmysterio** [comment]
One way or another, I am making a new version of my script generator and I want to make it based on the SRC modes and the SB library. Technically, I could generate ini files and offer them for integration. I just never worked with 3 and VC and I can mess up.

**Comment by wmysterio** [comment]
I am editing ini files. Progress is already small.

@x87 I noticed a small joint in "SASCM.ini (SRC)":
```text
0106=6,  locate_char_in_car_char_3d %1d% car %2d% radius %3d% %4d% %5d% sphere %6h%
```
There must be an actor:
```text
 %1d% cHar %2d%
```

**Comment by x87** [comment]
>There must be an actor:

corrected.

>Can the %8b:in-sphere\% record be replaced with %8b%?

yes, b: modifier is optional

**Comment by wmysterio** [comment]
@x87 Hi. I made preliminary versions of the INI files for the "gta3_scr", "gta_vc_scr" and even "gta_sa_scr_v2" modes. The description of the parameters was taken from the SB library.

Maybe you know a community that can check them out. I can fix bugs until the files are in working order.

The files do not yet contain opcodes from KLEO and other plugins, but this can be corrected later. Also, there are no parameters "m" and "o", I will change them later too.

[scr.rar](http://ru-script.3dn.ru/DELETE_LATER/scr.rar)

**Comment by x87** [comment]
Cool, I'll take a look

**Comment by wmysterio** [comment]
I have updated the archive. I made SCR INI files for GTA 3 and GTA VC (without plugins yet). I'll try to make the necessary folders for SB. People will already be able to do testing.
[scr](http://ru-script.3dn.ru/DELETE_LATER/scr.rar)

**Comment by x87** [comment]
This request has been fulfilled by adding new SBL edit modes in v4.0

**Comment by x87** [comment]
released in 4.0.0
